### DISCUSSION TITLE
-path and -screenshot dont work

### DISCUSSION THREAD

**Discussion by 1br00** [projectdiscovery/httpx#1127] [Ideas]
echo "testphp.vulnweb.com" | httpx -silent -path "listproducts.php?artist=<script>confirm(1)</script>" -mc 200 -follow-redirects -screenshot

doesn't work !

**Comment by ehsandeep** [comment]
@1br00 avoid creating duplicate issues/discussions, this is being discussed at https://github.com/projectdiscovery/httpx/issues/1125
